### PR TITLE
Add resnet50 for classification 

### PR DESCRIPTION
**Pull request by iwkkk**

### CODE REVIEW

File: cv/classification/resnet50/ddp_inference.sh
Comment (ShawnXuan) [inline cv/classification/resnet50/ddp_inference.sh:1]: rename to `infer.sh`
format this file like this:
```
python3 -m oneflow.distributed.launch \
  --nproc_per_node $GPU_NUMS \
  --master_addr 127.0.0.1 \
  --master_port $PORT  \
    ../main.py \
    --cfg ../configs/cnn_inference_settings.yaml \
    --model_arch $MODEL_ARCH
```

File: cv/classification/resnet50/ddp_training.sh
Comment (ShawnXuan) [inline cv/classification/resnet50/ddp_training.sh:1]: rename to train.sh
format this file

File: cv/classification/resnet50/ddp_inference.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export PYTHONPATH=$PWD:$PYTHONPATH
+set -aux
+
+GPU_NUMS=8
+PORT=12346
+MODEL_ARCH="resnet50"  # "swin_tiny_patch4_window7_224"
+
+python3 -m oneflow.distributed.launch --nproc_per_node $GPU_NUMS --master_addr 127.0.0.1 --master_port $PORT  \
+        ../main.py --cfg ../configs/cnn_inference_settings.yaml --model_arch $MODEL_ARCH
```
Comment (ShawnXuan) [inline cv/classification/resnet50/ddp_inference.sh]: main.py
Assuming the user will run bash resnet50/infer.sh under the directory cv/classification.

File: cv/classification/resnet50/ddp_training.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export PYTHONPATH=$PWD:$PYTHONPATH
+set -aux
+
+GPU_NUMS=8
+PORT=12346
+MODEL_ARCH="resnet50"  # "swin_tiny_patch4_window7_224"
+
+python3 -m oneflow.distributed.launch --nproc_per_node $GPU_NUMS --master_addr 127.0.0.1 --master_port $PORT  \
+        ../main.py --cfg ../configs/cnn_default_settings.yaml --model_arch $MODEL_ARCH  --epochs 50
```
Comment (ShawnXuan) [inline cv/classification/resnet50/ddp_training.sh]: main.py

File: cv/classification/resnet50/ddp_training.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export PYTHONPATH=$PWD:$PYTHONPATH
+set -aux
+
+GPU_NUMS=8
+PORT=12346
+MODEL_ARCH="resnet50"  # "swin_tiny_patch4_window7_224"
```
Comment (ShawnXuan) [inline cv/classification/resnet50/ddp_training.sh]: remove `# "swin_tiny_patch4_window7_224"`

File: cv/classification/resnet50/ddp_inference.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
+export PYTHONPATH=$PWD:$PYTHONPATH
+set -aux
+
+GPU_NUMS=8
+PORT=12346
+MODEL_ARCH="resnet50"  # "swin_tiny_patch4_window7_224"
```
Comment (ShawnXuan) [inline cv/classification/resnet50/ddp_inference.sh]: remove `# "swin_tiny_patch4_window7_224"`

File: cv/classification/resnet50/infer.sh [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+python3 -m oneflow.distributed.launch \
+    --nproc_per_node $GPU_NUMS \
+    --master_addr 127.0.0.1 \
+    --master_port $PORT  \
+    main.py \
+    --cfg configs/cnn_inference_settings.yaml \
+    --model_arch $MODEL_ARCH
+
+
```
Comment (zkyseu) [inline cv/classification/resnet50/infer.sh]: 一个小细节，多余的空行可以删掉

File: cv/classification/resnet50/train.sh [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+python3 -m oneflow.distributed.launch \
+    --nproc_per_node $GPU_NUMS \
+    --master_addr 127.0.0.1 \
+    --master_port $PORT  \
+    main.py \
+    --cfg configs/cnn_default_settings.yaml \
+    --model_arch $MODEL_ARCH \
+    --epochs 50
+
+
```
Comment (zkyseu) [inline cv/classification/resnet50/train.sh]: 多余空行可以删掉

**Review by zkyseu — COMMENTED** [review]
一个小细节可以修改一下，train.sh和infer.sh多余的空行可以删掉

**Review by zkyseu — APPROVED** [review]

**Pull request merged**